setup.py

```diff
@@ -5,8 +5,7 @@
     long_description = fh.read()
 
 requirements = [
-    #'wbuild @ git+https://github.com/gagneurlab/wBuild.git',
-    'wbuild>=1.7.1',
+    'wbuild @ git+https://github.com/gagneurlab/wBuild.git',
```

Comment:
why using the github repo instead of just the version? 

```suggestion
    'wbuild>=1.7.1',
```

Reply:
wbuild 1.7.1 via pip fails to install because it hasn't been packaged in a platform-independent way. I had a discussion with Leo and we settled on bumping wbuild with a working build in the next release